### PR TITLE
Kapacitor CLIs

### PR DESCRIPTION
**Pull request by sanderson**
Documents the command line tools associated with Kapacitor:

- `kapacitor`
- `kapacitord`

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/kapacitor/v1/reference/cli/kapacitord/version.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+```sh
+kapacitord version
+```
+
+## Examples
+
+```sh
+> kapacitord version
+Kapacitor OSS version v{{< latest-patch >}} (git: HEAD 00x0x0X0Xx0x0xXX0xxxXxx00000xX0000x0xXx0)
+```
```
Comment (jstirnaman) [inline content/kapacitor/v1/reference/cli/kapacitord/version.md:27]:
```suggestion
```sh
kapacitord version
```

The output is the following:

```sh
Kapacitor OSS version v{{< latest-patch >}} (git: HEAD 00x0x0X0Xx0x0xXX0xxxXxx00000xX0000x0xXx0)
```
```

**Review by jstirnaman — APPROVED** [review]
Overall, looks good - just a few suggestions. Thanks for taking this on!

File: content/kapacitor/v1/reference/cli/kapacitor/define-template.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Arguments
+
+- **template-id**: Unique identifier for the template
+
+## Flags {#command-flags}
+
+| Flag    | Description                        |
+| :------ | :--------------------------------- |
+| `-tick` | TICKscript filepath                |
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1/reference/cli/kapacitor/define-template.md:34]: Yeah... this CLI is showing its age.

File: content/kapacitor/v1/reference/cli/kapacitor/service-tests.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+## Arguments
+
+- **service**: Kapacitor service (handler) to execute service tests for
+
+## Examples
+
+```sh
+> kapacitor service-tests slack talk smtp
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1/reference/cli/kapacitor/service-tests.md:34]: I included the prompt to separate the command from the output. One thing I did elsewhere is put the output in an expand block, but I didn't do that here. Thoughts?

**Pull request merged**